python/desc/skycatalogs/catalog_creator.py

```diff
@@ -430,7 +433,7 @@ def create_galaxy_flux_catalog(self, config_file=None):
         from desc.skycatalogs import open_catalog, SkyCatalog
 
         self._gal_flux_schema = make_galaxy_flux_schema(self._logname)
-        BaseObject.get_bp_dir()
+        ###BaseObject.get_bp_dir()
```

Comment:
Any reason to not just delete this line?

Reply:
Nope.  There were a bunch more like that that I did delete.  Fixed in new commit.